js/join.js

```diff
@@ -410,3 +424,81 @@ submit.addEventListener('click', async () => {
 copyBtn.addEventListener('click', () => {
   navigator.clipboard.writeText(`Real Dev Squad Verification Link: ${url}`);
 });
+
+function countWords(textArea) {
+  let m;
+  const regex = /\w+/g;
+  let count = 0;
+  while ((m = regex.exec(textArea)) !== null) {
```

Comment:
what is the meaning of this line??
what is the meaning of m??

Reply:
In which ` while ((m = regex.exec(textArea)) !== null)` m is declared as a variable.
regex is defined as a regular expression pattern /\w+/g. This pattern matches one or more word characters (letters, digits, or underscores) in a global search.
m = regex.exec(textArea): This line executes the regular expression regex on the textArea and assigns the result to the variable m. The exec() method is used to search for matches of the regular expression pattern in the textArea.
(m = regex.exec(textArea)) !== null: The condition (m = regex.exec(textArea)) !== null checks if the result of the exec() method is not null. If null, it means that there are no more matches found in the textArea, and the loop will stop executing.